app/js/lib/foxbox/foxbox.js

```diff
@@ -223,24 +216,22 @@ export default class Foxbox extends Service {
       return Promise.resolve();
     }
 
-    const queryString = location.search.substring(1);
-    const searchParams = new URLSearchParams(queryString);
-
-    if (searchParams.has('session_token')) {
-      // There is a session token in the URL, let's remember it.
-      // @todo Find a better way to handle URL escape.
-      this[p.settings].session = searchParams.get('session_token')
-        .replace(/ /g, '+');
+    const url = new URL(location.href);
+    if (!url.searchParams.has('session_token')) {
+      return Promise.resolve();
+    }
 
-      // Remove the session param from the current location.
-      searchParams.delete('session_token');
-      location.search = searchParams.toString();
+    // There is a session token in the URL, let's remember it.
+    // @todo Find a better way to handle URL escape.
+    this[p.settings].session = url.searchParams.get('session_token')
+      .replace(/ /g, '+');
 
-      // Throwing here to abort the promise chain.
-      throw(new Error('Redirecting to a URL without session'));
```

Comment:
I removed throwing of error since I was so annoyed by constantly seeing it in the console log :) It seems that it's not really an error, but rather normal program flow. Tell me if you prefer to revert this change!